haaska.py

```diff
@@ -295,10 +295,12 @@ def entity_domain(x):
         return x['entity_id'].split('.', 1)[0]
 
     def is_supported_entity(x):
-        return entity_domain(x) in ha.config.allowed_domains
+        return entity_domain(x) in ha.config.exposed_domains
 
-    def is_skipped_entity(x):
+    def is_exposed_entity(x):
         attr = x['attributes']
+        if 'haaska_hidden' not in attr and 'hidden' not in attr:
+            return ha.config.expose_by_default
         return (('haaska_hidden' in attr and attr['haaska_hidden']) or
                 ('hidden' in attr and attr['hidden']))
```

Comment:
I think this logic might be a bit off and is requiring the opposite value (`haaska_hidden: true`) for it to be exposed properly.